src/ol/worker/loader.js

```diff
@@ -0,0 +1,43 @@
+import MVT from "../format/MVT";
+import RenderFeature from "../render/Feature";
+import OLWorker from './export';
+
+var blob;
+try {
+  blob = new Blob([OLWorker], {type: 'application/javascript'});
+} catch (e) { // Backwards-compatibility
+  blob = new MSBlobBuilder();
```

Comment:
The catch should not be necessary, IE10 seems to support standard Blob. See https://developer.mozilla.org/fr/docs/Web/API/Blob.